### ISSUE TITLE
[Bug] Main of eventscape.py yields error "not enough values to unpack (expected 8, got 5)"

### ISSUE THREAD

**Issue by maxoja** [ssantichaivekin/eMPRess#53]
To reproduce

1. `cd <repo>/empress/xscape `
2. `python eventscape.py` or `python -m empress.xscape.eventscape` after #6 is resolved
3. enter `out.csv`
4. enter `1`
5. enter `1`
6. enter `1`
7. enter `1`
8. Observe the error

In the first glance, this occurred because getInput.getInput(...) returns too few values than the caller expects and tries to unpack.

**Comment by ssantichaivekin** [comment]
This is because when we move and/or refactor things, we forgot to remove the main function in each file. I think that main function should be removed.

**Comment by maxoja** [comment]
If so, I have no other ways to validate the script for the time being, I think this issue should be now closed and a new issue should be summoned for the matter you mentioned.
